candle-transformers/src/models/mmdit/blocks.rs

```diff
@@ -250,6 +250,7 @@ impl ContextQkvOnlyJointBlock {
 
 // A QKV-attention that is compatible with the interface of candle_flash_attn::flash_attn
 // Flash attention regards q, k, v dimensions as (batch_size, seqlen, nheads, headdim)
+#[cfg(not(feature = "flash-attn"))]
```

Comment:
Would it be possible to keep flash attention optional in the same way it was before?

Reply:
Sure. I have passed down a `use_flash_attn` flag to the attention block, following the convention in UNet and other models elsewhere.